configs/altogether.yaml

```diff
@@ -0,0 +1,53 @@
+trainer:
+  precision: bf16-mixed
+  logger:
+  - class_path: lightning.pytorch.loggers.CSVLogger
+    init_args:
+      save_dir: /Users/Shinji/models
+  - class_path: lightning.pytorch.loggers.WandbLogger
+    init_args:
+      save_dir: /Users/Shinji/models
+  callbacks:
+  - class_path: lightning.pytorch.callbacks.LearningRateMonitor
+    init_args:
+      logging_interval: epoch
+  - class_path: lightning.pytorch.callbacks.EarlyStopping
+    init_args:
+      monitor: val_loss
+      min_delta: 0.0
+      patience: 10
+      verbose: true
+      mode: min
+  max_epochs: 100
+  max_time: 00:06:00:00
```

Comment:
remove max time example. Users may try to naievely do a 24 hr run.

Reply:
If someone changes it to 24, I assume they understand what's going on.